src/components/BudgetExpiryAlertAndModal/data/expiryThresholds.js

```diff
@@ -93,26 +93,26 @@ const expiryThresholds = {
   30: ({ intl, date }) => ({
     notificationTemplate: {
       title: intl.formatMessage({
-        id: 'adminPostal.learnerCredit.expiresInThirtyDaysNotification.title',
+        id: 'adminPortal.learnerCredit.expiresInThirtyDaysNotification.title',
         defaultMessage: 'Your Learner Credit plan expires in less than 30 days',
         description: 'Title for the notification that the Learner Credit plan is expiring in less than 30 days.',
       }),
       variant: 'danger',
       message: intl.formatMessage({
-        id: 'adminPostal.learnerCredit.expiresInThirtyDaysNotification.message',
+        id: 'adminPortal.learnerCredit.expiresInThirtyDaysNotification.message',
         defaultMessage: 'When your plan expires you will lose access to administrative functions and the remaining balance of your plan{apostrophe}s budget(s) will be unusable. Contact support today to renew your plan.',
         description: 'Message for the notification that the Learner Credit plan is expiring in less than 30 days.',
-      }, { aposrophe: "'" }),
+      }, { apostrophe: "'" }),
```

Comment:
[inform] I wonder if it's worth changing this up a bit to not treat `apostrophe` as a dynamic property passed into the message, given it is static.

For example, would this work?

```
{
  defaultMessage: "When your plan expires you will lose access to administrative functions and the remaining balance of your plan's budget(s) will be unusable. Contact support today to renew your plan.",
}
```